controllers/rollingupgrade_controller.go

```diff
@@ -319,10 +317,10 @@ func (r *RollingUpgradeReconciler) WaitForTermination(ruObj *upgrademgrv1alpha1.
 	return true, nil
 }
 
-func (r *RollingUpgradeReconciler) GetAutoScalingGroup(rollupName string) (*autoscaling.Group, error) {
-	val, ok := r.ruObjNameToASG.Load(rollupName)
+func (r *RollingUpgradeReconciler) GetAutoScalingGroup(namespacedName string) (*autoscaling.Group, error) {
+	val, ok := r.ruObjNameToASG.Load(namespacedName)
```

Comment:
Can you confirm that upgrades were tested after this change? Using the namespaced name in the `ruObjNameToASG` map makes sense. Just that `ruObjNameToASG` gets used at various places and need to ensure that those call sites are not making assumptions about the key in this map.

Reply:
The cache is updated with namespaced name: https://github.com/keikoproj/upgrade-manager/blob/97a1d4e4aebbaeed808cc096db936be00b4cbeba/controllers/rollingupgrade_controller.go#L453

I renamed variable name to clearly indicate that is a namespaced name.